### PR TITLE
Changelog

### PR DESCRIPTION
**Pull request by adam2392**
#### Reference issue
Bug fix change log for #7288

### CODE REVIEW

**Comment by larsoner** [comment]
That diff does not look correct

**Comment by adam2392** [comment]
> That diff does not look correct

@larsoner fixed.

File: doc/changes/latest.inc [hunk 1]
```diff
@@ -224,6 +224,8 @@ Bug
 
 - Fix placement of extrapolation points in :meth:`mne.Evoked.plot_topomap` and related functions when exactly three channels were used by `Mikołaj Magnuski`_.
 
+- Fix bug in reading annotations in :func:`read_annotations`, which would not accept ";" character.
```
Comment (agramfort) [inline doc/changes/latest.inc]: by who? please add name. thx

Comment (larsoner) [reply doc/changes/latest.inc]: Need to add the name here, I'll push a commit

**Comment by larsoner** [comment]
Killing Azure and Travis to save some cycles, just need to wait for CircleCI

**Comment by larsoner** [comment]
Thanks @adam2392

**Pull request merged**